### ISSUE TITLE
Thanks, i sucessfully found real x worm and your stormkitty/lumma in the files

### ISSUE THREAD

**Issue by Fuydutdtu** [d00mt3l/XWorm-5.6#4]
thanks!

**Comment by RedzHubzPS** [comment]
NVM

**Comment by capybarajtmc** [comment]
is this the real x worm?

**Comment by Fuydutdtu** [comment]
I closed it bc I was partially wrong, the storm kitty was bc of recovery.dll bc it uses same code to get passwords as storm kitty, but yes it is the real one just delete XWormV5.6Loader.exe and the rest is fine

**Comment by jytauth** [comment]
is the XWormV5.6Loader.exe rat? or if i open that i get ratted?

**Comment by Fuydutdtu** [comment]
> is the XWormV5.6Loader.exe rat? or if i open that i get ratted?

Yes, it is a Lumma payload, delete it and don't run it and everything else is ok to use

**Comment by jytauth** [comment]
> > is the XWormV5.6Loader.exe rat? or if i open that i get ratted?
> 
> Yes, it is a Lumma payload, delete it and don't run it and everything else is ok to use

bruh i ran it before, what will happen? how do i delete it

**Comment by Sudolmao** [comment]
its a rootkit redownload windows

**Comment by Sudolmao** [comment]
> > > is the XWormV5.6Loader.exe rat? or if i open that i get ratted?
> > 
> > 
> > Yes, it is a Lumma payload, delete it and don't run it and everything else is ok to use
> 
> bruh i ran it before, what will happen? how do i delete it

i would redownload windows tho you can check if your ratted by seeing if you can factory reset the pc from the setting if it says recovery media not found immediatly watch https://www.youtube.com/watch?v=mTDbHgs9dHk&pp=ygUkaG93IHRvIHJlaW5zdGFsbCB3aW5kb3dzIDExIGZyb20gdXNi

**Comment by namo-dot** [comment]
it wont load for me

**Comment by NFS11** [comment]
is this safe or not. everyone saying the loader is a grabber but everything else chill. is that so? i plan on testing and showcasing this, thanks.

**Comment by hahaxaxa44** [comment]
@Fuydutdtu can u help me download and install it on discord

**Comment by Yxpanti** [comment]
@hahaxaxa44 i can help you
add .yzpanti
